lib/package.rb

```diff
@@ -27,7 +27,6 @@ def require_gem(gem_name_and_require = nil, require_override = nil)
 
```

Comment:
We also don't need to define `require_gem` in `lib/package.rb` anymore, as a regular require should work here as highline will have been installed before we get here.